src/test/java/org/tarantool/ClientReconnectIT.java

```diff
@@ -213,11 +214,54 @@ public void run() {
         });
     }
 
+    // DO NOT REMOVE THIS TEST
+    // Motivation: this test checks start/stop correctness
+    // of TarantoolControl class which is used by other tests.
+    // This test is commented out because the class is used
+    // for internal purposes only and isn't related to
+    // the connector testing.
+    //    @Test
+    //    @DisplayName("follow up the issue #164")
+    //    void testStartStopTarantoolInstance() throws InterruptedException {
+    //        int numberOfParallelInstances = 4;
+    //        CountDownLatch finished = new CountDownLatch(numberOfParallelInstances);
+    //        List<String> instancesNames = new ArrayList<>(numberOfParallelInstances);
+    //
+    //        for (int i = 0; i < numberOfParallelInstances; i++) {
+    //            String instance = "startStop" + (i + 1);
+    //            instancesNames.add(instance);
+    //            control.createInstance(
+    //                instancesNames.get(i),
+    //                LUA_FILE,
+    //                makeInstanceEnv(3301 + i + 1, 3321 + i + 1)
```

Comment:
It is easy to catch 'Address already in use' with larger number of threads, because jdk-testing use 3313 as a consule port. I would set 3401 and 3501 here.